reverbePrintemps/spiced
# Basic calculation functions

# Context

This PR aims to tackle [the basic calculation functions challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-functions-2/challenges-js-functions-2.md#calculations).

# Changes

- [x] Implement `add` function
- [x] Implement `subtract` function
- [x] Implement `multiply` function
- [x] Implement `divide` function
- [x] Implement `remainder` function

# How to test

1. Clone repo
2. Checkout this branch
3. Run `node index.js`
4. There should be no assertion errors in the console

# TODO


# GIF

![](https://media.giphy.com/media/gEvab1ilmJjA82FaSV/giphy-downsized.gif)
